### PR TITLE
Fix some 404's

### PR DESCRIPTION
**Pull request by DeanmvSG**
Some links have been moved so a quick update to them

### CODE REVIEW

File: marketplace_builder/views/pages/how-platformos-works/basic-concepts.liquid [hunk 1]
```diff
@@ -15,7 +15,7 @@ searchable: true
 
 **Partials** (partial templates) allow you to easily organize and reuse your code by extracting pieces of code to their own files. They help you improve code readability and follow the principle of DRY (Don’t Repeat Yourself). You can parameterize partials and use them in various places, e.g. layouts, pages, Authorization Policies, Form Configurations.
 
-[Learn how to use pages, layouts and partials in our Get Started guides](/get-started/pages)
+[Learn how to use pages, layouts and partials in our Get Started guides](get-started/pages/pages)
```
Comment (pavelloz) [inline marketplace_builder/views/pages/how-platformos-works/basic-concepts.liquid:18]: I think leading slash wasnt a mistake

Comment (DeanmvSG) [reply marketplace_builder/views/pages/how-platformos-works/basic-concepts.liquid:18]: @pavelloz looking at other links, you have it in some pages (e.g. related topics at the bottom of this page - https://documentation.platform-os.com/get-started/pages/pages).  But I have done it in some links and not others in this commit.   Let me know what you'd/ @diana-lakatos  would like and I'll update 😄

Comment (diana-lakatos) [reply marketplace_builder/views/pages/how-platformos-works/basic-concepts.liquid:18]: @DeanmvSG Thank you for fixing these links! The leading slash is needed when we add links, we only omit them for slugs.

**Comment by pavelloz** [comment]
TY :-)

**Pull request merged**